reminders/tests.py

```diff
@@ -82,3 +82,10 @@ def test_reminder_with_invalid_time(self, participant0, date_time=TIME_0):  # no
     def test_invalid_reminder_exist_twice(self, persist_reminder):
         with pytest.raises(Exception, match=EXIST_REMINDER_ERROR):
             persist_reminder.save()
+
+    def test_get_earliest_reminder(self):
+        expected_reminder = Reminder.objects.get(
+            participant_id__event_id__title="event1",
+            participant_id__user_id__username="testUser1"
+        )
+        assert expected_reminder == Reminder.objects.get_next_reminder()
```

Comment:
Can you add some tests for checking the sending of the email method?

Reply:
Also add please some tests for the notification functionality 😃

Reply:
1. Yes I can add some tests to ensure that the emails are sent to the end-user.
2. @danninon, can you add some tests for your next PR to check the site notification.